lib/checks/navigation/meta-refresh-evaluate.js

```diff
@@ -1,8 +1,25 @@
-function metaRefreshEvaluate(node, options, virtualNode) {
-  var content = virtualNode.attr('content') || '',
-    parsedParams = content.split(/[;,]/);
+const separatorRegex = /[;,\s]/
+const validRedirectNumRegex = /^[0-9.]+$/
 
-  return content === '' || parsedParams[0] === '0';
+function metaRefreshEvaluate(node, options, virtualNode) {
```

Comment:
Do we want `export default function metaRefreshEvaluate` if it's the only function in the file?